src/main/kotlin/no/nav/familie/ba/sak/behandling/vedtak/vedtaksperiode/VedtaksperiodeService.kt

```diff
@@ -87,6 +90,27 @@ class VedtaksperiodeService(
         return vedtaksperiodeRepository.finnVedtaksperioderFor(vedtakId = vedtak.id)
     }
 
+    fun oppdaterFortsattInnvilgetPeriodeMedAutobrevBegrunnelse(vedtak: Vedtak,
+                                                               vedtakBegrunnelseSpesifikasjon: VedtakBegrunnelseSpesifikasjon) {
+        val vedtaksperioder = hentPersisterteVedtaksperioder(vedtak)
+
+        val fortsattInnvilgetPeriode: VedtaksperiodeMedBegrunnelser? = vedtaksperioder.firstOrNull()
+
+        if (fortsattInnvilgetPeriode == null || vedtaksperioder.size > 1)
+            throw Feil("Finner ingen eller flere vedtaksperioder ved fortsatt innvilget")
```

Comment:
```suggestion
        val fortsattInnvilgetPeriode: VedtaksperiodeMedBegrunnelser = vedtaksperioder.singleOrNull() ?: throw Feil("Finner ingen eller flere vedtaksperioder ved fortsatt innvilget")
```

veldig flisespikkeri og har ingenting å si på funksjonalitet

Reply:
singleOrNull var ny 👏

Reply:
La inn denne. Bra tips!